iscservice/demo/libdemo.c

```diff
@@ -170,6 +173,46 @@ int compute_minmax(enum op op, struct m0_isc_comp_private *pdata,
 	return M0_FSO_AGAIN;
 }
 
+int compute_minmax2(enum op op, struct m0_isc_comp_private *pdata,
+		    struct m0_buf *out, int *rc)
+{
+	int               i;
+	double           *p;
+	m0_bcount_t       len;
+	struct mm_result  res = {};
+	struct m0_buf     buf = M0_BUF_INIT0;
+
+	len = m0_isc_io_res((struct m0_stob_io *)pdata->icp_data, &(char*)p);
+	if (len < 0) {
+		*rc = M0_ERR_INFO((int)len, "failed to read data");
+		return M0_FSO_AGAIN;
+	}
+
```

Comment:
Do you like:
M0_ASSERT(len % 8 == 0);
?
It is obviously true, but this ensures that no boundary cutting happens.

Reply:
Good idea, added. Thanks!